streams/src/main/java/org/apache/kafka/streams/processor/assignment/TaskAssignmentUtils.java

```diff
@@ -16,14 +16,63 @@
  */
 package org.apache.kafka.streams.processor.assignment;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
 import java.util.SortedSet;
+import java.util.UUID;
+import java.util.stream.Collectors;
 import org.apache.kafka.streams.processor.TaskId;
+import org.apache.kafka.streams.processor.assignment.KafkaStreamsAssignment.AssignedTask;
+import org.apache.kafka.streams.processor.internals.assignment.Graph;
+import org.apache.kafka.streams.processor.internals.assignment.MinTrafficGraphConstructor;
+import org.apache.kafka.streams.processor.internals.assignment.RackAwareGraphConstructor;
+import org.apache.kafka.streams.processor.internals.assignment.RackAwareGraphConstructorFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * A set of utilities to help implement task assignment via the {@link TaskAssignor}
  */
 public final class TaskAssignmentUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(TaskAssignmentUtils.class);
+
+    private TaskAssignmentUtils() {}
+
+    /**
+     * Return a "no-op" assignment that just copies the previous assignment of tasks to KafkaStreams clients
+     *
+     * @param applicationState the metadata and other info describing the current application state
+     *
+     * @return a new map containing an assignment that replicates exactly the previous assignment reported
+     *         in the applicationState
+     */
+    public static Map<ProcessId, KafkaStreamsAssignment> identityAssignment(
+        final ApplicationState applicationState
+    ) {
```

Comment:
Kafka Streams formatting: first parameter always on the same line (no matter how ridiculously long it gets)

```suggestion
    public static Map<ProcessId, KafkaStreamsAssignment> identityAssignment(final ApplicationState applicationState) {
```